tools/make/tests.toml

```diff
@@ -4,19 +4,24 @@
 
 # This is a cargo-make file included in the toplevel Makefile.toml
 
+[tasks.check-all-features-chunked]
+description = "Build all permutations of all features for the provided chunk index (3 chunks)"
+category = "ICU4X Development"
+install_crate = { crate_name = "cargo-all-features", binary = "cargo-check-all-features", test_arg = ["--help"] }
+command = "cargo"
+args = ["check-all-features", "--n-chunks", "3", "--chunk", "${@}"]
+
 [tasks.check-all-features]
 description = "Build all permutations of all features"
 category = "ICU4X Development"
 install_crate = { crate_name = "cargo-all-features", binary = "cargo-check-all-features", test_arg = ["--help"] }
-install_crate_args = ["--version", "^1.9"]
```

Comment:
Question: why did you remove the `install_crate_args`; IIRC it helps prevent cases where people have old versions of the tool installed

Reply:
oh, yeah, Rob's changes changed them and I removed them because they were referencing a hash. I'll reintroduce them

Reply:
oh, yeah, Rob's changes changed them and I removed them because they were referencing a hash. I'll reintroduce them